scripts/validate

```diff
@@ -15,7 +15,7 @@ echo Verifying modules
 go mod verify
 
 echo Generating files
-go generate
+#go generate
```

Comment:
Why is the `go generate` commented out here?

Reply:
commented for internal testing, uncommented and push the changes in [b535de2](https://github.com/rancher/rke/pull/2963/commits/b535de26f2aeeb169aac045698db067d51403f99)